### PR TITLE
detect/cip: check for sccanf return-value - v2

### PR DESCRIPTION
**Pull request by 0xEniola**
Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket: https://redmine.openinfosecfoundation.org/issues/6753

Previous PR: #10346

Describe changes:
- Fix `clang-format` error.

### CODE REVIEW

**Review by jufajardini — CHANGES_REQUESTED** [review]
The fix itself makes sense to me, thanks :)

Could you please add more context in the commit message about why was the fix needed (like how was this issue found), and, if you think it's need, why is the change a good solution?

Also, please Leave the ticket number as the last thing on the commit message, as this is the preferred format for our commit messages :)

File: src/detect-cipservice.c [hunk 1]
```diff
@@ -157,8 +157,9 @@ static DetectCipServiceData *DetectCipServiceParse(const char *rulestrc)
             goto error;
         }
 
-        sscanf(token, "%2" SCNu8, &var);
-        input[i++] = var;
+        if (sscanf(token, "%2" SCNu8, &var) == 1) {
```
Comment (victorjulien) [inline src/detect-cipservice.c:160]: what does it mean if sscanf returns another value? What should we do with the token then?

**Comment by 0xEniola** [comment]
> The fix itself makes sense to me, thanks :)
> 
> Could you please add more context in the commit message about why was the fix needed (like how was this issue found), and, if you think it's need, why is the change a good solution?
> 
> Also, please Leave the ticket number as the last thing on the commit message, as this is the preferred format for our commit messages :)

OK! Understood :).

File: src/detect-cipservice.c (hunk 1 above)
Comment (catenacyber) [reply src/detect-cipservice.c:160]: Looks right as done a few lines above

Comment (0xEniola) [reply src/detect-cipservice.c:160]: OK! Thank you :).

I am also curious, I tried to search what `%2` formats to, but I could not find any answers.

Comment (catenacyber) [reply src/detect-cipservice.c:160]: `"%2" SCNu8` 

because `SCNu8` is a portable macro to scan a uint8_t, so it would look like `"%2c"` in the end

Comment (catenacyber) [reply src/detect-cipservice.c:160]: yes please

**Pull request closed without merging**